themes/theme-rui/src/components/Multiselect.styles.ts

```diff
@@ -1,7 +1,7 @@
 import { ThemeComponent, cva } from '@marigold/system';
 
 export const MultiSelect: ThemeComponent<'MultiSelect'> = {
-  field: cva('grid gap-y-0.5'),
+  field: cva('w-full space-y-2'),
```

Comment:
the w-full must be removed otherwise the property will not work, or its is not working correctly in storybook 🤔